### ISSUE TITLE
Bug when updating from 0.118 to 0.119.

### ISSUE THREAD

**Issue by spapas** [EmilStenstrom/django-components#855]
Hey friends, I've got the following code in one of my components:

```
{% component "collapsible-card" "Στοιχεία συντάκτη / εκκινητή διαδικασίας "|add:app.get_initiator_details header_color="bg-info" header_text_color="text-white" border_color="border-info" %}
```

This worked fine on 0.118. When upgrading to 0.119 I get:

```
django.template.exceptions.TemplateSyntaxError: Could not find variable at start of |add:app.get_initiator_details.
```

This can be fixed using a {% with %} however I'd rather avoid changing all my components.

My understanding is that this bug was introduced because of https://github.com/EmilStenstrom/django-components/pull/827. Can you please rollback these breaking changes? 

TIA

**Comment by EmilStenstrom** [comment]
@spapas thanks for reporting! I’d say a better plan to fix this is to add a test for this case (it’s likely missing, and then fix the bug. The fix you are referring to solves other problems that we still want. Could you write up a small test case that we can then fix?

**Comment by spapas** [comment]
Hello @EmilStenstrom the PR of the test case needed is here https://github.com/EmilStenstrom/django-components/pull/856

**Comment by JuroOravec** [comment]
Hi, @spapas for reporting this. It was my bad, I forgot consider Django's template filters in the last refactor.  I wrote fix in https://github.com/EmilStenstrom/django-components/pull/857

**Comment by JuroOravec** [comment]
This has been released in [v0.121](https://github.com/EmilStenstrom/django-components/releases/tag/0.121) :)
